### PR TITLE
Fixed html totum physics

### PR DESCRIPTION
**Pull request by jakezira**

### CODE REVIEW

File: type_templates/html.js [hunk 1]
```diff
@@ -170,9 +170,9 @@ export default e => {
   let staticPhysicsIds = [];
   {
     const physicsId = physics.addBoxGeometry(
-      new THREE.Vector3(),
-      new THREE.Quaternion(),
-      new THREE.Vector3(width * scale * app.scale.x / (app.scale.x * 2), height * scale * app.scale.y / (app.scale.y * 2), 0.001),
+      object.position,
```
Comment (avaer) [inline type_templates/html.js:173]: Does this need to be `matrixWorld` for nesting support?

Comment (jakezira) [reply type_templates/html.js]: Updated

File: type_templates/html.js [hunk 2]
```diff
@@ -169,10 +169,18 @@ export default e => {
   let physicsIds = [];
   let staticPhysicsIds = [];
   {
+    object.matrixWorld.decompose(localVector, localQuaternion, localVector2);
+    localVector2.multiply(
+      new THREE.Vector3(
```
Comment (avaer) [inline type_templates/html.js:174]: Why not be consistent and use local?

Comment (jakezira) [reply type_templates/html.js]: Fixed the bad memory code

**Review by avaer — APPROVED** [review]

**Pull request merged**